### DISCUSSION TITLE
Why does a patch release gets hightlited in red?

### DISCUSSION THREAD

**Discussion by stoivo** [pnpm/pnpm#4856] [Q&A] [answered]
Why does a patch release gets hightlited in red?
![Screenshot 2022-06-05 at 14 05 15](https://user-images.githubusercontent.com/3492040/172049596-e79c50a0-3508-49b5-81b6-a66efe3a756d.png)

I get it that it's 17 patch releases but that should not matte for semver release system?

pnpm --version  7.1.7

**Comment by mcmxcdev** [comment, marked as answer]
If you are referring to esbuild in the list, it is because all version changes for packages that haven't reached v1 yet are considered potentially breaking.

Only starting with v1 and higher, semver has to be applied strictly.
